Project/Assets/ML-Agents/Examples/Startup/Scripts/Startup.cs

```diff
@@ -7,24 +7,48 @@ namespace MLAgentsExamples
     internal class Startup : MonoBehaviour
     {
         const string k_SceneVariableName = "SCENE_NAME";
+        private const string k_SceneCommandLineFlag = "--scene-name";
 
         void Awake()
         {
-            var sceneName = Environment.GetEnvironmentVariable(k_SceneVariableName);
+            var sceneName = "";
+            
+            // Check for the CLI '--scene-name' flag.  This will be used if
+            // no scene environment variable is found.
+            var args = Environment.GetCommandLineArgs();
+            Console.WriteLine(String.Join(" ", args));
```

Comment:
Remove logging?

Reply:
I was thinking this could be useful for making sure arguments were passed correctly in the logs, since the arguments passed come via some misdirection from `mlagents-learn`.  What do you think about leaving it?

Reply:
In that case, maybe preface it with "Command line args: " or something like that?

Reply:
Done.